quest-config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "AzureDevOps": {
+        "Org": "msft-skilling",
+        "Project": "Content",
+        "AreaPath": "Production\\Digital and App Innovation\\DotNet and more\\MAUI"
```

Comment:
You could also use `\\dotnet` as the last component here. It's up to you.

Reply:
```suggestion
        "AreaPath": "Production\\Digital and App Innovation\\DotNet and more\\dotnet"
```